app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.UserProfileFragment">
+
+
+    <TextView
+        android:id="@+id/settings"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="35dp"
+        android:layout_marginTop="90dp"
+        android:text="@string/settings_title"
+        android:textAppearance="@style/TextAppearance.AppCompat.Display1"
```

Comment:
What about changing text size instead of text appearance?

Reply:
Or, if you want to use the whole style, `@style/TextAppearance.Material3.DisplayLarge` might be better